index.rst

```diff
@@ -42,20 +42,151 @@
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. _introduction:
+
+Introduction
+============
+
+At the summit it is sometimes desirable for instrument data to be analyzed rapidly after data acquisition in order to get quick feedback of data quality.
+The different use cases for processing data are described in :cite:`DMTN-111` and in this document we focus on data processing directly related to data acquisition.
+
+Data processing must be triggered in some way with enough information to be able to work out which data to process and which analysis pipeline to use.
+The unique identifier for every observation is the "Observation Identifier" (aka "obsid" aka "image name") that is guaranteed by the observatory systems to be unique for every observation regardless of telescope (main telescope vs AuxTel), instrument, or which controller was involved to take the data.
+This obsid is issued by the camera control system whenever data are acquired.
+Additionally, there is a "grouping" concept that can be used by scripts to indicate that observations taken with the same group identifier are related [*]_.
+The group ID is how standard visits will be observed with both snaps being part of the same group.
+The group ID will be provided to each script by the ScriptQueue and will be unique for that execution of the script.
+
+Since DM software always mediates data access via a Data Butler this leads to a requirement that raw data must always be retrievable from a butler with a Data ID solely containing the obsid, and optionally, the detector name or group name.
```

Comment:
Shouldn't we be able to retrieve snaps using a Data ID of the group ID and snap number, with no obsid provided?
Capitalization of "Data ID" or "data ID" should be made consistent.

Reply:
On ingest the butler doesn't know the snap number so it can't be used in a Data ID. I'm not sure how can work one out if you send takeImages with more than one exposure request vs calling takeImages in a loop and passing in a counter.

Reply:
In this paragraph I'm trying to say that we must always be able to retrieve using the obsid regardless of anything else. I'm not removing the possibilities of using other parameters.